aten/src/ATen/NamedTensorUtils.h

```diff
@@ -10,6 +11,11 @@ inline bool has_names(TensorList tensors) {
       tensors.begin(), tensors.end(), [](const Tensor& t) { return t.is_named(); });
 }
 
-}
+namespace namedinference {
+
+optional<std::vector<Dimname>> reduction_op(optional<DimnameList> self_names, int64_t dim);
```

Comment:
Is there a better name we could use for this function? As a helper function it's not what I'd usually think of as being reduction operator.

Reply:
Maybe something like `namedinference::remove_dimension`

Reply:
yeah that would be fine. reduce_op to mean just feels like it implies an actual tensor reduction operator like sum or mean

Reply:
Done. I named it `namedinference::erase_name`.